Resources/config/services.xml

```diff
@@ -5,26 +5,44 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
+        <service id="Shivas\VersioningBundle\Formatter\FormatterInterface"
+                 alias="Shivas\VersioningBundle\Formatter\GitDescribeFormatter"/>
+        <service id="Shivas\VersioningBundle\Writer\WriterInterface"
+                 alias="Shivas\VersioningBundle\Writer\VersionWriter"/>
```

Comment:
these should be marked as private explicitly, to be private even on 3.4 (rather than half private for the BC layer)